RecoJets/JetProducers/plugins/InputGenJetsParticleSelector.cc

```diff
@@ -198,6 +198,7 @@ InputGenJetsParticleSelector::ResonanceState InputGenJetsParticleSelector::fromR
           return kDirect;
         if (!isExcludedFromResonance(id))
           break;
+        [[fallthrough]];
```

Comment:
This seems to be correct, but please @rappoccio @knash @gkasieczka confirm

Reply:
This one looks correct to me.